src/static/riot/competitions/detail/submission_manager.tag

```diff
@@ -1,55 +1,199 @@
 <submission-manager>
     <h1>Submission manager</h1>
+    <div if="{ opts.admin }" style="padding-bottom: 20px;">
+        <a class="ui green button" href="{csv_link}">
+            <i class="icon download"></i>Download as CSV
+        </a>
+        <div class="ui dropdown blue button">
+            <i class="icon redo"></i>
+            <div class="text">Rerun all submissions per phase</div>
+            <div class="menu">
+                <div class="header">Select a phase</div>
+                <div class="item" each="{phase in opts.competition.phases}" onclick="{rerun_phase.bind(this, phase)}">{ phase.name }</div>
+            </div>
+        </div>
+    </div>
+    <div class="ui icon input">
+        <input type="text" placeholder="Search..." ref="search" onkeyup="{ filter }">
+        <i class="search icon"></i>
+    </div>
+    <select if="{opts.admin}" class="ui dropdown" ref="phase" onchange="{ filter }">
+        <option value="">Phase</option>
+        <option value=" ">-----</option>
+        <option each="{ phase in opts.competition.phases }" value="{ phase.id }">{ phase.name }</option>
+    </select>
+    <select class="ui dropdown" ref="status" onchange="{ filter }">
+        <option value="">Status</option>
+        <option value=" ">-----</option>
+        <option value="Cancelled">Cancelled</option>
+        <option value="Failed">Failed</option>
+        <option value="Finished">Finished</option>
+        <option value="Preparing">Preparing</option>
+        <option value="Running">Running</option>
+        <option value="Scoring">Scoring</option>
+        <option value="Submitted">Submitted</option>
+        <option value="Submitting">Submitting</option>
+    </select>
     <table class="ui celled selectable inverted table">
         <thead>
         <tr>
             <th>#</th>
             <th>File name</th>
+            <th if="{ opts.admin }">Owner</th>
+            <th if="{ opts.admin }">Phase</th>
             <th class="right aligned" width="50px">Status</th>
+            <th class="center aligned" if="{ opts.admin }">Actions</th>
             <th class="right aligned" width="50px">Leaderboard?</th>
         </tr>
         </thead>
         <tbody>
-        <tr each="{ submission in submissions }">
-            <td>1</td>
+        <tr each="{ submission in submissions }" onclick="{ show_modal.bind(this, submission) }" class="clickable">
+            <td>1</td> <!-- TODO how is this number supposed to increment? -->
```

Comment:
I think if you do `<tr each="{ submission, index in submissions}"` it will give you the index, then `index + 1` will be the count?

https://riot.js.org/guide/#loops

```
The array elements need not be objects. They can be strings or numbers as well. 
In this case you need to use the { name, i in items } construct as follows:

<my-tag>
  <p each="{ name, i in arr }">{ i }: { name }</p>

  this.arr = [ true, 110, Math.random(), 'fourth']
</my-tag>

The name is the name of the element and i is the index number. Both of these 
labels can be anything that’s best suited for the situation.
```